android/app/src/main/res/values/splash.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color type="color" name="splash_background">#ffffff</color>
```

Comment:
We could move this one to `colors.xml`, just to have everything in the same place

Reply:
The thing is that we override colors per jurisdiction 😬

Reply:
That makes sense then 💯